receiver/k8sclusterreceiver/README.md

```diff
@@ -266,7 +266,7 @@ spec:
       serviceAccountName: otelcontribcol
       containers:
       - name: otelcontribcol
-        image: otelcontribcol:latest # specify image
+        image: otel/opentelemetry-collector-contrib-dev:latest # specify image
```

Comment:
Ideally, the recommendation should be the official release instead of the dev image.

```suggestion 
        image: otel/opentelemetry-collector-contrib
```